openedx/core/djangoapps/content/course_overviews/migrations/0014_courseoverview_certificate_available_date.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
I think you're ok here, but just in case you haven't seen this yet: https://openedx.atlassian.net/wiki/spaces/OpenDev/pages/23003228/Everything+About+Database+Migrations